### PR TITLE
small refactoring class Netutil and tests added

### PR DESCRIPTION
**Pull request by moh-sushi**
Hi,

small refactoring of class `jodd.io.NetUtil`:
- rename method  'validateHostIp' to `validateAgaintIPAdressV4Format`
- update javadoc

Tests added, too.

Bye,
Sascha

### CODE REVIEW

File: jodd-core/src/main/java/jodd/io/NetUtil.java [hunk 1]
```diff
@@ -101,15 +99,20 @@ public static boolean isSocketAccessAllowed(int localIp, int socketIp, int mask)
 	}
 
 	/**
-	 * Validates IP address given as a string.
+	 * checks given string against ip address v4 format
+	 *
+	 * @param input an ip address - may be null
+	 * @return <tt>true</tt> if param has a valid ip v4 format <br/><tt>false</tt> otherwise
```
Comment (igr) [inline jodd-core/src/main/java/jodd/io/NetUtil.java]: compilation fails because of <br/> :)))

**Comment by igr** [comment]
Now it works :)

File: jodd-core/src/main/java/jodd/io/NetUtil.java (hunk 1 above)
Comment (moh-sushi) [reply jodd-core/src/main/java/jodd/io/NetUtil.java]: I've seen it - fix committed :-)

**Comment by igr** [comment]
Like the naming of the method, very clear now!

**Pull request merged**